modules.d/90kernel-modules-extra/module-setup.sh

```diff
@@ -179,8 +180,16 @@ installkernel() {
         | (
             cd "$depmod_module_dir" || exit
             xargs -r realpath -se --
-        ) \
-        | instmods || return 1
+        ) 2>&1 \
+        | while read mod; do
+            if [[ "$mod" == "realpath: "* ]]
+            then
+                prwarning "Find invalid file in modules.dep."
```

Comment:
Non-existing module entry in modules.dep.

Reply:
The message is updated.